Makefile

```diff
@@ -0,0 +1,50 @@
+FEATURES_MINIMAL = --no-default-features --features 'std cargo' -p clap:3.0.0-beta.5
+FEATURES_FULL = --features 'wrap_help yaml regex unstable-replace unstable-multicall unstable-grouped'
+
```

Comment:
One of the things I did in #2901 was document [design guidelines for people to keep in mind](https://github.com/clap-rs/clap/pull/2901/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R1-R5).  Should we bring those over and add any for the documentation role this file fills (like "only include non-CI features that are intended to document the CI for contributors")?

I know the original comments are of a little lower value since we aren't calling into this from CI as much as we are in this PR.